content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -4,7 +4,7 @@ In a regular Git workflow, <abbr>DVC repository</abbr> versions are typically
 synchronized among team members. And [DVC Experiments] are internally connected
 to this commit history, so you can similarly share them.
 
-## Basic workflow: store as peristent commits
+## Basic workflow: store as persistent commits
```

Comment:
This is an unrelated fix, oops. Found it when looking for places to link from (but didn't end up linking from this file).